### ISSUE TITLE
[feat] Enable Deleting Events for Specific Dates

### ISSUE THREAD

**Issue by Saurabhchaudhary9799** [Code-Social/official-website#411]
**Is your feature request related to a problem? Please describe.**
I'm frustrated when I can't delete specific events from a selected date in the app. Currently, there’s no option to remove individual events by date, which makes it challenging to manage or update my schedule accurately without deleting all events or manually clearing each one.

**Describe the solution you'd like**
I'd like an option to delete a specific event on any chosen date directly from the event list or calendar view. This feature should allow users to quickly locate an event on a particular date and remove it, ensuring the calendar remains organized and up-to-date.

**Comment by Saurabhchaudhary9799** [comment]
@Rizwan102003 , could you please assign this to me ?

**Comment by Saurabhchaudhary9799** [comment]
> @Rizwan102003 , could you please assign this to me ?

I want to wok on this issue , please assign me ?

**Comment by Rizwan102003** [comment]
@Saurabhchaudhary9799 i don't think this will be an useful feature. Please look into the rest of the issues to work on.
